In-house patch to remove unnecessary dependencies from Heat's
requirements files. The specific reasons are as follows:

dnspython3	Not applicable to Solaris

--- swift-2.7.0/requirements.txt.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/requirements.txt	2016-08-08 17:21:52.277804965 -0700
@@ -2,12 +2,10 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 
-dnspython>=1.12.0;python_version<'3.0'
-dnspython3>=1.12.0;python_version>='3.0'
+dnspython>=1.12.0
 eventlet>=0.17.4  # MIT
 greenlet>=0.3.1
 netifaces>=0.5,!=0.10.0,!=0.10.1
 pastedeploy>=1.3.3
 six>=1.9.0
 xattr>=0.4
-PyECLib>=1.2.0                          # BSD
--- swift-2.7.0/swift.egg-info/requires.txt.~1~	2016-03-25 02:44:40.000000000 -0700
+++ swift-2.7.0/swift.egg-info/requires.txt	2016-08-08 17:20:12.178152517 -0700
@@ -4,10 +4,4 @@ netifaces>=0.5,!=0.10.0,!=0.10.1
 pastedeploy>=1.3.3
 six>=1.9.0
 xattr>=0.4
-PyECLib>=1.2.0
-
-[:(python_version<'3.0')]
 dnspython>=1.12.0
-
-[:(python_version>='3.0')]
-dnspython3>=1.12.0
